tuplex/core/include/Partition.h

```diff
@@ -69,6 +69,7 @@ namespace tuplex {
         void loadFromFile(const URI& uri);
 
         int64_t                 _numRows;
+        int64_t                 _numSkip; // number of rows to skip, currently only used at the output (Result set)
```

Comment:
let's not add this to Partition

Reply:
Is there an alternative way to earse beginning rows from a partition? Here, I want to transform the partition during the output writing to remove some front rows, like `[1, 2, ..., 3, 4].map(x -> x + 1).take(bottom=2)` will compute to `[2, 3, .., 4, 5]` and I want to trim it to just `[4, 5]`. We can push this logic to `resultSet` too, but that will make `resultSet` more complex